process-engine/process_engine_meta
# 🐛 Feature/support feature linking

## What did you change?

I added the `--trust-local-modules` flag of minstall, so that when we change the version of a local-module-dependency to a non-semver-version like `feature~some_feature_name`, minstall will assume that the associated local module fulfills that dependency-version, even though it is non-semver and can therefore not be checked

## How can others test the changes?

- Change any essential-projects or process-engine dependency to a non-semver version
- Run `npm install` without this fix
- See how it tries to install that non-semver version from npm
- Run `npm install` with this fix
- See how now the associated local module gets linked, instead of npm trying to install it.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
